src/Loader.php

```diff
@@ -121,14 +121,29 @@ public static function is_dev() {
 	}
 
 	/**
-	 * Gets an array of enabled WooCommerce Admin features/sections.
+	 * Gets a build configured array of enabled WooCommerce Admin features/sections.
 	 *
-	 * @return bool Enabled Woocommerce Admin features/sections.
+	 * @return array Enabled Woocommerce Admin features/sections.
 	 */
 	public static function get_features() {
 		return apply_filters( 'woocommerce_admin_features', array() );
```

Comment:
@rrennick What do you think about just using this existing filter instead of introducing a new one? You'd have to make sure it runs after ours that populates the data. Is that a deal breaker?

Reply:
@psealock in my experience trying to work with that filter, it would only work if done during a build. Though my memory could be foggy - but I believe that is the big difference between that, and @rrennick's approach here on a more run-time filter that can work in a built version of the plugin.

Reply:
> in my experience trying to work with that filter, it would only work if done during a build.

After I added that filter I found it could only disable the Devdocs screen dynamically.